static/js/messages.js

```
@@ -0,0 +1,291 @@
+// MessagesManager - Singleton for handling both message pane and toast notifications
+export class MessagesManager {
+  constructor() {
+    if (MessagesManager.instance) {
+      return MessagesManager.instance;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)